src/pages/instances/InstanceDetailPanelContent.tsx

```diff
@@ -85,6 +89,20 @@ const InstanceDetailPanelContent: FC<Props> = ({ instance }) => {
             {settings?.environment?.server_clustered ? instance.location : "-"}
           </td>
         </tr>
+        <tr>
+          <th className="u-text--muted">Root storage pool</th>
+          <td>
+            <Link
+              to={`/ui/project/${instance.project}/storage/pool/${(instance.expanded_devices?.root as LxdDiskDevice).pool}`}
+            >
+              {(instance.expanded_devices?.root as LxdDiskDevice).pool}
```

Comment:
Better assign a variable `const rootPool = ...` and reuse it, like we do for the pid.

Also the assumption here, that the root disk always has the name `root` is not correct. We should add a function to pick the root device by filtering all devices by the ones that are
1. type=disk
2. path=/
3. source is not set

Maybe put this into a function, so we can use it on the overview tab and in the side panel.

Reply:
I understand now. I have extrapolated this into a function now, please let me know what you think :)